src/data-dictionary/events/K8sPodSample/nodeName.md

```diff
@@ -0,0 +1,9 @@
+---
+name: nodeName
```

Comment:
nodeName is another one that overlaps multiple K8s events. It's a good example of one we could have a single attribute entry for.